### PR TITLE
Document deps

### LINKED ISSUE #1025: Update installation instruction for system libs: libgomp1, etc.
Hey guys,

I am new to this project so I apologies in advanced if I have run into a simple error. I did try searching before creating a ticket. I wanted to test out the pretrained models on some audio files. I fired up a fresh VPS and installed the python package. Here are the steps I took, and the error I got:

1. Fire up fresh Ubuntu 17.04 VPS
2. Install PIP
3. Install deepspeech
4. Download deepspeech-0.1.0-models.tar.gz
5. Extract the files in the tar - this gives me the following: https://i.imgur.com/vFhO6mk.png
6. Run the command `deepspeech output_graph.pb my_audio.wav alphabet.txt lm.binary trie`
7. I get the following error:

image

Thanks

### PR DESCRIPTION
**Pull request by lissyx**
Fixes #1025

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ pip install deepspeech-gpu
 deepspeech output_model.pb my_audio_file.wav alphabet.txt
 ```
 
-See the output of `deepspeech -h` for more information on the use of `deepspeech`.
+See the output of `deepspeech -h` for more information on the use of `deepspeech`. Please check [required runtime dependencies](native_client/README.md#required-dependencies).
```
Comment (kdavis-mozilla) [inline README.md]: Maybe rephrase this slightly. Something like...

...of `deepspeech`. (If you experience problems running `deepspeech`, please check [required runtime dependencies](native_client/README.md#required-dependencies).)

**Comment by lissyx** [comment]
Doc only, we can merge now.
